### ISSUE TITLE
 Error: Request failed with status code 504

### ISSUE THREAD

**Issue by jetLuoxianjie** [zerodytrash/TikTok-Live-Connector#194]
Error: Request failed with status code 504

**Comment by bakteribaik** [comment]
same here, so the first until second try to connect get `Error: Request failed with status code 504` but when the third connection is said isConnected: true, but it cant read chat and cant read statusEnd event.

**Comment by fairuzrahman** [comment]
i got this also, found any solution? @bakteribaik ?

**Comment by bakteribaik** [comment]
> i got this also, found any solution? @bakteribaik ?

No, i think that server side error. Im not use this library for now

**Comment by fairuzrahman** [comment]
> > i got this also, found any solution? @bakteribaik ?
> 
> No, i think that server side error. Im not use this library for now

hmm any lib similar to this?
